wrapper/src/test/java/integration/refactored/container/tests/ReadWriteSplittingTests.java

```diff
@@ -516,4 +733,18 @@ public void test_transactionResolutionUnknown() throws SQLException {
       }
     }
   }
+
+  protected void assertFirstQueryThrows(Connection connection, String expectedSQLErrorCode) {
```

Comment:
I believe these methods are also in AuroraFailoverTest. To remove duplication, can we move the ones from AuroraFailoverTest to AuroraTestUtility and then them from here as well as AuroraFailoverTest? Can we also remove queryInstanceId from this file and use AuroraTestUtility#queryInstanceId instead?

Reply:
yes, sounds good!